enclave/enclave.go

```diff
@@ -65,3 +84,15 @@ func CreateAzureAttestationToken(data []byte, url string) (string, error) {
 	}
 	return internal.CreateAzureAttestationToken(report, data, url)
 }
+
+// AttestOption	configures an attestation function.
+type AttestOption struct {
+	apply func(*internal.Options)
+}
+
+// WithIgnoreTCBStatus ignores an invalid TCB level.
+//
+// Callers must verify the TCBStatus field in the report themselves.
+func WithIgnoreTCBStatus() AttestOption {
+	return AttestOption{func(o *internal.Options) { o.IgnoreErr = attestation.ErrTCBLevelInvalid }}
+}
```

Comment:
This could be moved to a separate package to reduce code duplication with `eclient/eclient.go`.
Might negatively impact usability though, so I think this could also be kept as is

Reply:
Yes, I also considered this. And there was a further problem: I wanted AttestOption to be a struct without exported fields so users don't create dependencies on the current implementation.